devtools/aesh/pom.xml

```diff
@@ -20,24 +20,85 @@
         <maven.compiler.source>1.8</maven.compiler.source>
     </properties>
 
+    <!-- not used atm
```

Comment:
will it ever be used ?

Reply:
ATM I'm directly calling the Gradle binary or wrapper for compile/dev-mode. I noticed the gradle-api, but it is not on mavencentral. I commented it out for this PR as I would like to play more with that api to see if it's possible to call Gradle native instead of starting a separate process.

Reply:
@stalep with this, is it possible to use the gradle api?